pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -46,6 +46,9 @@ type VPCNetworkConfigurationSpec struct {
 	// +kubebuilder:validation:MaxLength=8
 	// +optional
 	ShortID string `json:"shortID,omitempty"`
+	// Policy path of the NSX VPC the Namespace associated with.
+	// +optional
+	VPCPath string `json:"vpcPath,omitempty"`
```

Comment:
VpcPath?

Reply:
VPC is the abbreviation of Virtual Private Cloud, we always use VPC but not Vpc to present this meaning, just like NSX. So I don't think we shall use camel style for it.

Reply:
VPCPath +1

Reply:
As discussed in google chat， Could we change to ”VPC“？

Reply:
Sure, will update.